src/components/Dropdown/Dropdown.tsx

```diff
@@ -308,19 +364,44 @@ export default class Dropdown extends AutoControlledComponent<
   private renderItemsList(
     styles: ComponentSlotStylesInput,
     variables: ComponentVariablesInput,
-    getMenuProps: (options?: GetMenuPropsOptions, otherOptions?: GetPropsCommonOptions) => any,
-    getItemProps: (options: GetItemPropsOptions<ShorthandValue>) => any,
     isOpen: boolean,
     highlightedIndex: number,
+    toggleMenu: () => void,
+    selectItemAtIndex: (index: number) => void,
+    getMenuProps: (options?: GetMenuPropsOptions, otherOptions?: GetPropsCommonOptions) => any,
+    getItemProps: (options: GetItemPropsOptions<ShorthandValue>) => any,
+    getInputProps: (options?: GetInputPropsOptions) => any,
   ) {
-    const accessibilityMenuProps = getMenuProps({ refKey: 'innerRef' })
+    const accessibilityMenuProps = {
+      ...getMenuProps({ refKey: 'innerRef' }, { suppressRefError: true }),
+    }
+    if (!this.props.search) {
+      const accessibilityInputProps = getInputProps()
+      accessibilityMenuProps['aria-activedescendant'] =
+        accessibilityInputProps['aria-activedescendant']
+      accessibilityMenuProps['onKeyDown'] = e => {
+        this.handleListKeyDown(
+          e,
+          highlightedIndex,
+          accessibilityInputProps['onKeyDown'],
+          toggleMenu,
+          selectItemAtIndex,
+        )
+      }
+    }
     const { innerRef, ...accessibilityMenuPropsRest } = accessibilityMenuProps
 
     return (
-      <Ref innerRef={innerRef}>
+      <Ref
+        innerRef={(listNode: HTMLElement) => {
+          this.listNode = listNode
+          innerRef(listNode)
+        }}
+      >
         <List
           {...accessibilityMenuPropsRest}
           styles={styles.list}
+          tabIndex={-1}
           aria-hidden={!isOpen}
```

Comment:
I am thinking how we can move such accessibility things to behaviors. At least for configuring attributes, also based on props

Reply:
we may need to have a meeting at some point, as this is one of many things accessibility related that we need to encapsulate.